xAPI.md

```diff
@@ -507,6 +507,47 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
+This section explains and shows graphically how different pieces of xAPI can fit together.  
+
+##### Tracking Experiences Through Systems
+
+![Data Flow in xAPI](/xAPIDataFlow.jpg)
+
+Figure 1 shows the tracking of experiences through the data flow of xAPI. A user either accesses on their own, or 
+is provided access to web-enabled content.  The user, known by the LRS, is tracked in the experiences that he or 
+she has while taking that content. A Data Analysis Tool, acting as a Service with permission, can grab the data  
```

Comment:
Instead of sentences 2 and 3: 

A learner experiences a learning activity and data about that experience is tracked to the Learning Record Store (LRS). Authentication with the LRS may use the learners credentials, the activity provider's credentials or a combination.